code/index.html

```diff
@@ -1,12 +1,95 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <style>
+      @import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+      </style>
+      
     <meta charset="utf-8" />
-    <title>Project Name</title>
-    <!-- dont forget the link to the css file -->
-    <!-- dont forget the meta tag for responsiveness -->
+    <title>contact us</title>
+<link rel="stylesheet" href="style.css">    <!-- dont forget the meta tag for responsiveness -->
   </head>
   <body>
-    <!-- Build your site up as you want it! Make sure to use the form element and try to send that form to httpbin by adding action="https://httpbin.org/anything" and method="POST" to your form tag. -->
+
+    <form action="https://httpbin.org/anything" method="POST">
+  
+<form>
+  <h2>The Cat Working Space</h2>
+  <h3> Erstelle deinen Account</h3>
+  <fieldset>
+  <label>
+        <span>Name</span>
+        <input type="text" name="name" placeholder="Vorname" />
```

Comment:
When an element has several attributes, it's nice to give them new lines for better readability:
```
<input
  type="text"
  name="name"
  placeholder="Vorname" />
```